### PR TITLE
Fix for issue #16.

### LINKED ISSUE #16: Add new Google Cloud Platform connection fails
When I try to add a service endpoint in VSTS, after supplying the required info and JSON key, it fails with the following error message:
screen shot 2017-09-04 at 21 40 48
Not sure what else might be relevant info, feel free to reach me to provide more details.

### PR DESCRIPTION
**Pull request by ILMTitan**
Fixes #16.

### CODE REVIEW

File: container-build-task/task.json [hunk 1]
```diff
@@ -7,9 +7,9 @@
   "description": "Build a Docker image with Google Cloud Container Builder.",
   "helpMarkDown": "Use [Google Cloud Container Builder](https://cloud.google.com/container-builder/docs) to build a Docker image and place it in [Google Container Registry](https://cloud.google.com/container-registry/docs).",
   "version": {
-    "Major": 0,
+    "Major": 1,
```
Comment (ivannaranjo) [inline container-build-task/task.json:10]: Can we not update to a major version of 1? This sort of implies that this is a GA version.

**Review by ivannaranjo — CHANGES_REQUESTED** [review]
Can we not update the major version?

File: container-build-task/task.json (hunk 1 above)
Comment (ILMTitan) [reply container-build-task/task.json:10]: Unfortunately, VSTS will auto-upgrade tasks that are incremented by less than a Major version. To enable selecting different versions, we have to upgrade the Major version. Fortunately, the task versions are only visible when using the task, and the extension as a whole (the version number visible on the extension page) is still just 0.0.4, indicating a prerelease version.

**Comment by ILMTitan** [comment]
We have to update the major version of the tasks.
The extension version as a whole is still just 0.0.4 (aka a prerelease version).

**Review by ivannaranjo — APPROVED** [review]

**Comment by ivannaranjo** [comment]
LGTM

**Pull request merged**